src/components/dropdown/dropdown.js

```diff
@@ -634,6 +636,8 @@ class Dropdown extends React.Component {
         klass += ` ${className}--selected`;
       }
 
+      /* eslint-disable jsx-a11y/mouse-events-have-key-events */
+      /* eslint-disable jsx-a11y/click-events-have-key-events */
```

Comment:
are we intending to disable these now and fix them later? or forever disable them?

Reply:
The explanation for the rule changes are explained in the commits, I tried to keep each commit atomic to each change I was making. I can add code comments too if that would be useful. 

https://github.com/Sage/carbon/pull/1520/commits/5655452c5e61618cc7592130993c91ed8f6806cc

Reply:
ok cool - a comment in the code might be useful too 👍 

Suggests that a future refactor may be useful in which each item can manage it's own events instead of the parent element.